liquibase-standard/src/main/java/liquibase/database/core/EnterpriseDBDatabase.java

```diff
@@ -58,4 +59,9 @@ public String getDefaultDriver(String url) {
     public Integer getDefaultPort() {
         return 5444;
     }
+
+    @Override
+    public boolean supportsCreateIfNotExists(Class<? extends DatabaseObject> type) {
```

Comment:
EnterpriseDBDatabase doesn't support the if NOT EXISTS statement, but because it inherits from PostgresDatabase, it's set to true.
Fix this as well.